src/aktualizr_lite/main.cc

```diff
@@ -108,6 +108,7 @@ static std::unique_ptr<Uptane::Target> find_target(const std::shared_ptr<SotaUpt
 }
 
 static int do_update(LiteClient &client, Uptane::Target &target) {
+  target.InsertEcu(client.primary_ecu.first, client.primary_ecu.second);
```

Comment:
It means that aktualizr-lite supports updating just a single ECU. If an update of multiple ECUs is required then ECUs (or hardware IDs) has to be specified in the targets.json and Target object respectively.

Reply:
yep - this is currently limited to a single primary ECU. I think we could untangle this logic to do secondaries, but i just haven't had anyone ask me for it.

Reply:
ok